app/controllers/miq_request_controller.rb

```diff
@@ -79,7 +121,20 @@ def show_list
     @sortdir = session[:request_sortdir].nil? ? "ASC" : session[:request_sortdir]
     @no_checkboxes = true # Don't show checkboxes, read_only
     kls = @layout == "miq_request_ae" ? AutomationRequest : MiqRequest
-    @view, @pages = get_view(kls, :named_scope => prov_scope(user_options(params)))
+    require 'byebug'
+    if session[:request_filters].nil?
+      scope = user_options(params)
+    else
+      scope = {
+        :reason_text => session[:request_filters][:reasonText],
+        :applied_states => session[:request_filters][:approvalStates],
+        :type_choice => session[:request_filters][:types],
+        :user_choice => session[:request_filters][:selectedUser],
+        :time_period => session[:request_filters][:selectedPeriod],
+      }
+    end
```

Comment:
maybe move them to a new private method like `def named_scope_data(session[:request_filters]) `and return the scope.